site/src/pages/community/articles.mdx

```diff
@@ -141,6 +141,17 @@ Send a pull request by editing [this page](https://github.com/line/armeria/edit/
 
 ### Slides and videos
 
+<Article
+  title="Spring BootユーザのためのArmeria入門"
+  url="https://matsumana.info/blog/2020/07/30/introduce-to-armeria-for-spring-users/"
+  author="松﨑学"
+  date="20200730"
+  type="slides"
+  description="
+    TBW
```

Comment:
Thank you for adding!
Could you update with the following?

```
Spring Bootユーザ向けに、以下の内容について説明されています。
・Armeriaが提供する機能、その中でもSREの観点で特に嬉しい機能
・ArmeriaをSpring BootとIntegrationして使う場合のコード例
・Spring Bootで実装されている既存のサービスへのArmeria導入シナリオ例
```

Reply:
Updated :bow: